setup.py

```diff
@@ -1,19 +1,30 @@
-from setuptools import setup
+import os
 
+from setuptools import setup, find_packages
 
-def readme():
-    with open('README.rst') as f:
+
+exec(open('auth_backends/_version.py').read())
+
+
+def read(*paths):
+    """Build a file path from `paths` and return the contents."""
+    with open(os.path.join(*paths), 'r') as f:
         return f.read()
 
 
 setup(
     name='edx-auth-backends',
-    version='0.1',
+    version=__version__,
     description='Custom edX authentication backends and pipeline steps',
-    long_description=readme(),
+    long_description=(
+        read('README.rst') + '\n\n' +
+        read('HISTORY.rst') + '\n\n' +
+        read('AUTHORS')
+    ),
     classifiers=[
         'Development Status :: 4 - Beta',
         'License :: OSI Approved :: GNU Affero General Public License v3',
+        'Programming Language :: Python',
```

Comment:
Works for me!

On Fri, Feb 20, 2015 at 1:46 PM, Renzo Lucioni <notifications@github.com>
wrote:

> In setup.py
> <https://github.com/edx/auth-backends/pull/3#discussion_r25092230>:
>
> >      classifiers=[
> >          'Development Status :: 4 - Beta',
> >          'License :: OSI Approved :: GNU Affero General Public License v3',
> > +        'Programming Language :: Python',
>
> It's seems to be a convention to list the 'Programming Language :: Python'
> classifier, followed by classifiers for each supported Python version.
> There are no hard and fast rules here, and I wanted to be thorough.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/auth-backends/pull/3/files#r25092230>.
>